app/views/admin/shared/_tabbed_navbar.erb

```diff
@@ -0,0 +1,20 @@
+<% nested_menu_items = AdminMenu.get_nested_menu_items(request.path.split("/")[-2], request.path.split("/")[-1]) %>
+
+<% if nested_menu_items[:children].any?%>
+
+  <header class="mb-3">
+    <h2 class="crayons-title"><%= nested_menu_items[:name].to_s.titleize %></h2>
+    <div class="block s:flex items-center space-between">
+      <nav class="crayons-tabs crayons-tabs--wrapped">
+        <% nested_menu_items[:children].each do |item| %>
+          <a
+            href="/admin/<%= request.path.split("/")[- 2] %>/<%= item[:controller] %>"
```

Comment:
```suggestion
            href="/admin/<%= request.path.split("/")[-2] %>/<%= item[:controller] %>"
```

But again, this request mangling directly in a view makes me a bit uneasy. Is there a way we can do this in the controller instead?

Reply:
I moved this into the admin helper where we have a function that deduces the scope and controller name based on the request path ✅